server/src/services/comment.service.ts

```diff
@@ -88,6 +91,50 @@ export class CommentService {
     });
   }
 
+  public async getMentionedNotifications(userId: string, isRead = false) {
+    const notifications = await this.commentUsersRepository.find({
+      where: {
+        userId,
+        isRead,
+      },
+      order: {
+        createdAt: 'DESC',
+      },
+      relations: ['comment'],
+    });
+
+    if (!notifications) {
+      throw new NotFoundException('User notifications not found');
+    }
+
+    try {
+      const _notifications = notifications.map(async (notification) => {
+        const [, commentLink] = await this.getAppLinks(notification.comment.appVersionsId, notification.comment);
+        const user = await this.usersRepository.findOne({
+          where: { id: notification.comment.user.id },
+          relations: ['avatar'],
+        });
+        const creator = {
+          firstName: user.firstName,
+          lastName: user.lastName,
+          avatar: user.avatar?.data.toString('base64'),
+        };
+        return {
+          id: notification.id,
+          creator,
+          comment: notification.comment.comment,
+          createdAt: notification.comment.createdAt,
+          updatedAt: notification.comment.updatedAt,
+          commentLink,
+          isRead: notification.isRead,
+        };
+      });
+      return Promise.all(_notifications);
```

Comment:
should use await to resolve all `_notifications`

Reply:
async/await implicitly returns promise,

```
return Promise.all(_notifications);
```
is same as 
```
return await Promise.all(_notifications);
```